jaxb-ri/xjc/src/main/java/com/sun/tools/xjc/addon/accessors/PluginImpl.java

```diff
@@ -71,6 +89,58 @@ public boolean run( Outline model, Options opt, ErrorHandler errorHandler ) {
                     Logger.getLogger(PluginImpl.class.getName()).log(Level.SEVERE, null, ex);
                 }
             }
+            Iterator<JFieldVar> flds = co.implClass.fields().values().iterator();
+            while (flds.hasNext()) {
+            	try {
+            		JFieldVar fld = flds.next();
+            		String fldName = fld.name();
+            		Set<JAnnotationUse> annotsCopy = new HashSet<JAnnotationUse>(fld.annotations());
+            		String getterName = "get" + fldName.substring(0, 1).toUpperCase() + fldName.substring(1);
```

Comment:
Can there be "is" instead of "get" for booleans?

Reply:
You won't find appropriate getters if you are searching by the wrong name. Consider following property:
```java
private boolean positive;
public boolean isPositive();
public setPositive(boolean positive);
```
In that case it will skip moving annotations from field to method.